.circleci/config.yml

```diff
@@ -1221,6 +1221,11 @@ jobs:
           name: Create and upload incident type, incident field, mapper and a layout to Cortex XSOAR
           command: python3 Tests/sdknightly/create_entities_for_nightly_sdk.py HelloWorld --artifacts-folder $CIRCLE_ARTIFACTS
       - *upload_entities_to_cortex_xsoar
+      - run:
+          name: Zip and upload a pack to the Cortex XSOAR's marketpalce
+          command: |
+            printf "y\nZipPackTest\nZipPackTest\n1\n1\ny\ny\nZipPackTest\ny\n6.0.0" | demisto-sdk init -n ZipPackTest
```

Comment:
Can you explain what you're printing here?

Reply:
Since the init command is interactive command (need input for integration name, version etc) printf print those arguments and pipe them to the init command
need also comment in the config file itself?